src/docs/faq.ts

```diff
@@ -65,7 +65,7 @@ export const faqColTwo: faqCol = [
 
     question: "When can I apply?",
     answer:
-      "Sign up at our application center before March 23th 11:59 PM to make sure you don't miss out!",
+      "Sign up at our application center starting February 2nd and before March 8th 11:59 PM to make sure you don't miss out!",
```

Comment:
Might be helpful to say that these dates are all 2026 since they're being added so early, just in case people get confused and think the dates are from last edition